Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -194,7 +196,7 @@ foreach ($Folder in $FolderList) {
     }
     # If the file doesn't exist Add that to the bad folder list -- means the folder is being scanned
     else {
-        Write-SimpleLogfile -String ("[FAIL] - Possible AV Scanning: " + $FilePath) -name $LogFile -OutHost
+        Write-SimpleLogfile -String ("[FAIL] - Possible AV Scanning: " + $Folder) -name $LogFile -OutHost
```

Comment:
Shouldn't this be the full path?

Reply:
the fullpath shows:
[7/11/2022 11:55:50 AM] - [FAIL] - Possible AV Scanning: c:\program files\microsoft\exchange server\v15\fip-fs\eicar.bat
And folder shows:
[7/11/2022 12:09:57 PM] - [FAIL] - Possible AV Scanning: c:\program files\microsoft\exchange server\v15\fip-fs

I think it is better to show only the folder that is not configured and not including the file that we are using to test it because it doesn't care to the user.